docs/source/WritingTests.rst

```diff
@@ -82,6 +82,10 @@ Additionally, we'll save a raw copy of the whiteboard contents on a file named
 (maybe you want to use the result of a benchmark directly with your custom made scripts
 to analyze that particular benchmark result).
 
+If you need to attach several output files, you can also use
+``self.outputdir``, which points to the ``test-results/data``
```

Comment:
Well it's `$job_results/test-results/$test_id/data` which I shrunk to `test-results/data` (I wanted to emphasize it's basically the test's results root + `data`, but let me put there the full path to avoid confusion.